jovian/utils/commit.py

```diff
@@ -52,37 +53,38 @@ def commit(message=None,
+        filename(string, optional): The filename of the current Jupyter notebook or Python script. This is
+            detected automatically in most cases, but in certain environments like Jupyter Lab or password protected notebooks, the detection
             may fail and the filename needs to be provided using this argument.
 
 
-        project(string, optional): Name of the |Jovian| project to which the current notebook/file should 
-            be committed. Format: 'username/title' e.g. 'aakashns/jovian-example' or 'jovian-example' 
-            (username of current user inferred automatically). If the project does not exist, a new one is 
-            created. If it exists, the current notebook is added as a new version to the existing project, if 
-            you are a owner/collaborator. If left empty, project name is picked up from the `.jovianrc` file in the 
-            current directory, or a new project is created using the filename as the project name. 
+        project(string, optional): Name of the |Jovian| project to which the current notebook/file should
+            be committed. Format: 'username/title' e.g. 'aakashns/jovian-example' or 'jovian-example'
+            (username of current user inferred automatically). If the project does not exist, a new one is
+            created. If it exists, the current notebook is added as a new version to the existing project, if
+            you are a owner/collaborator. If left empty, project name is picked up from the `.jovianrc` file in the
+            current directory, or a new project is created using the filename as the project name.
 
-        new_project(bool, optional): Whether to create a new project or update the existing one. Allowed option 
+        new_project(bool, optional): Whether to create a new project or update the existing one. Allowed option
             are False (use the existing project, if a .jovianrc file exists, if available), True (create a new project)
 
-        git_commit(bool, optional): If True, also performs a Git commit and records the commit hash. This is 
+        git_commit(bool, optional): If True, also performs a Git commit and records the commit hash. This is
             applicable only when the notebook is inside a Git repository.
 
         git_message(string, optional): Commit message for git. If not provided, it uses the `message` argument
 
     .. attention::
-        Pass notebook's name to `filename` argument, in certain environments like Jupyter Lab and password protected 
+        Pass notebook's name to `filename` argument, in certain environments like Jupyter Lab and password protected
         notebooks sometimes it may fail to detect notebook automatically.
     .. |Jovian| raw:: html
 
         <a href="https://jovian.ml/?utm_source=docs" target="_blank"> Jovian.ml </a>
     """
     global _current_slug
+    _current_slug = get_current_slug()
```

Comment:
This causes duplication of the value of `_current_slug` in 2 files: slug.py and commit.py. This can cause problems, as the two values may go out of sync. 

Avoid this duplication and have a single source of truth.